### DISCUSSION TITLE
关于bakaXL的联机宣传片里有PCL2，我急匆匆一看啥也没有这件事

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#2804] [提问] [answered]
[就这玩意](https://www.bilibili.com/video/BV1FC4y1G7aw)
lm之前说后期对接协议，是还没对接好吗？

**Comment by SqrtTan** [comment]
~快点最好，受不了了~

**Comment by Pigeon0v0** [comment]
首先这不是 BakaXL 官方的视频，其次确实会对接，但没那么快，请等待官方消息

**Comment by WForst-Breeze** [comment, marked as answer]
BakaXL 现已于 2023 年 10 月 8 日将其部分服务与 Octo 章鱼网络进行对接，并对其可用性进行初步评估，待评估完成后，Octo 网络会全面对其他启动器开放。但请注意，Octo 开放不等价于 PCL 上线联机，具体信息以龙猫发布的消息为准。

**Reply by SqrtTan** [reply]
懂了
